source/containers.tex

```diff
@@ -856,6 +856,24 @@
 then the operations described below
 are implemented by constexpr functions.
 
+\indexlibrarymember{array}{operator<=>}%
+\indexlibrarymember{deque}{operator<=>}%
+\indexlibrarymember{forward_list}{operator<=>}%
+\indexlibrarymember{list}{operator<=>}%
+\indexlibrarymember{vector}{operator<=>}%
+\indexlibrarymember{map}{operator<=>}%
+\indexlibrarymember{set}{operator<=>}%
+\indexlibrarymember{multiset}{operator<=>}%
+\indexlibrarymember{multimap}{operator<=>}%
+\indexlibrarymember{unordered_map}{operator<=>}%
+\indexlibrarymember{unordered_set}{operator<=>}%
+\indexlibrarymember{unordered_multiset}{operator<=>}%
+\indexlibrarymember{unordered_multimap}{operator<=>}%
```

Comment:
```suggestion
\indexcont{operator<=>}%
```

Reply:
Readded `forward_list`.

Reply:
The difference is that `\indexcont` uses `\indexlibrarymemberx`, which only adds the member under the container, rather than both ways.